src/Differ/DiffConsoleFormatter.php

```diff
@@ -87,7 +87,7 @@ function ($matches) {
 
                         return sprintf($lineTemplate, $string);
                     },
-                    Preg::split('#\R#u', $diff)
+                    preg_split('#\R#', $diff)
```

Comment:
please add a warning to not change this into `Preg::...` and why

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4107